main/app/reactiveLayer/allostaticController/conf/default.ini

```diff
@@ -1,5 +1,5 @@
 [ALLOSTATIC]
-drives		(narration exploration demonstration)
+drives		()

```

Comment:
I don't think this should be changed, otherwise we change the behavior of the year 2 demo.